### PR TITLE
Feature/solution99

### PR DESCRIPTION
**Pull request by rikba**
Can you please have a look at this awesome solution? @FloCF

### CODE REVIEW

File: Python/Problem033.ipynb [hunk 1]
```diff
@@ -81,7 +81,7 @@
    ]
   }
  ],
- "metadata": {
+ "Ketadata": {
```
Comment (FloCF) [inline Python/Problem033.ipynb:84]: Passt mir nicht...

File: Python/Problem099.ipynb [hunk 2]
```diff
@@ -0,0 +1 @@
+# Was ist die Lösung?
```
Comment (FloCF) [inline Python/Problem099.ipynb:1]: Okay...

**Review by FloCF — COMMENTED** [review]
Alles scheiße!

**Comment by rikba** [comment]
:cry:

**Pull request closed without merging**